### DISCUSSION TITLE
[Projects Beta] Milestone column values should link directly to milestone pages

### DISCUSSION THREAD

**Discussion by robin-aws** [community/community#10234] [Projects and Issues]
Hi there! 

Just a minor UX suggestion: I notice that if you click a milestone entry in the Milestones column, it leads to a target like https://github.com/org/repo/milestones/Name%20Of%20Milestone. 

This sort of gets the job done since you can see the set of issues/PRs under the milestone, but linking directly to to the milestone page with a target like https://github.com/org/repo/milestone/42 would be better, as that page also includes the milestone due date and description.

**Comment by Shinigami92** [comment]
Exactly this :+1: 

Also this row should be clickable

![image](https://user-images.githubusercontent.com/7195563/150019748-b2534cc5-f990-4c6d-b8ee-826e81fa0f9c.png)

And should then just link directly to https://github.com/faker-js/faker/milestone/1

**Comment by 700software** [comment]
> linking directly to to the milestone page with a target like https://github.com/org/repo/milestone/42 would be better, as that page also includes the milestone due date and description.

This is a simple change and would be greatly helpful. We are interfacing with non-developers and need to provide transparency to meaning of milestones via the milestone description.
